include/boost/geometry/util/series_expansion.hpp

```diff
@@ -0,0 +1,1099 @@
+// Boost.Geometry
+
+// Contributed and/or modified by Adeel Ahmad.
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_GEOMETRY_UTIL_SERIES_EXPANSION_HPP
+#define BOOST_GEOMETRY_UTIL_SERIES_EXPANSION_HPP
+
+#include <boost/geometry/util/math.hpp>
+
+namespace boost { namespace geometry { namespace series_expansion {
+
+    /*
```

Comment:
there is a lot of duplication in the maxima scripts inside comments, could you try to decrease it?

Reply:
I just removed the duplicated code in [this](https://github.com/boostorg/geometry/pull/486/commits/cc19342b4eac30183fd6f40d4d4d0a8b36fed885) commit. Please let me know if I should further reduce the comments.

Reply:
> Also the following script (taken from area formula, right?) is repeated 7 times!
...

Yes, it matches the one from area formula, except the last part `s/epsCT(2)/eps2/g;`. Before that the script was converting `eps2` -> `epsCT(2)`. So, I will just keep a single copy of this script and remove the duplicated ones.

> Finally, is the script taken from http://geographiclib.sourceforge.net/html/geod.mac without modifications? If so you can just add the link in the comments.

The scripts are indeed taken from (http://geographiclib.sourceforge.net/html/geod.mac), however, I had to make minor adjustments e.g. change the function header and data types to keep it consistent with Boost Geometry guidelines. Is it okay to discard those changes? If so, I will only provide a link to `geod.mac`.

Reply:
>The scripts are indeed taken from (http://geographiclib.sourceforge.net/html/geod.mac), however, I had to make minor adjustments e.g. change the function header and data types to keep it consistent with Boost Geometry guidelines. Is it okay to discard those changes? If so, I will only provide a link to geod.mac, otherwise, I will add them in a separate file.

Ok, since they contain changes, I think it is useful to keep them in Boost.Geometry.

Reply:
I have moved the Maxima scripts to: [doc/other/maxima/geod.mac](https://github.com/BoostGSoC18/geometry/blob/feature/geodesic_direct/doc/other/maxima/geod.mac).